GliaWidgets/Public/Glia/Glia+StartEngagement.swift

```diff
@@ -55,53 +69,7 @@ extension Glia {
         )
     }
 
-    /// Starts the engagement.
-    ///
-    /// - Parameters:
-    ///   - engagementKind: Engagement media type.
-    ///   - configuration: Engagement configuration.
-    ///   - queueID: Queue identifier.
-    ///   - theme: A custom theme to use with the engagement.
-    ///   - sceneProvider: Used to provide `UIWindowScene` to the framework. Defaults to the first active foreground scene.
-    ///
-    /// - throws:
-    ///   - `GliaCoreSDK.ConfigurationError.invalidSite`
-    ///   - `GliaCoreSDK.ConfigurationError.invalidEnvironment`
-    ///   - `GliaCoreSDK.ConfigurationError.invalidRegionEndpoint`
-    ///   - `GliaCoreSDK.ConfigurationError.invalidSiteApiKey`
-    ///   - `GliaError.engagementExists`
-    ///
-    public func start(
-        _ engagementKind: EngagementKind,
-        configuration: Configuration,
-        queueID: String,
-        theme: Theme = Theme(),
-        features: Features = .all,
-        sceneProvider: SceneProvider? = nil
-    ) throws {
-        let completion = { [weak self] in
-            try self?.startEngagement(
-                engagementKind: engagementKind,
-                theme: theme,
-                features: features,
-                sceneProvider: sceneProvider
-            )
-        }
-        do {
-            try configure(
-                with: configuration,
-                queueId: queueID
-            ) {
-                try? completion()
-            }
-        } catch GliaError.configuringDuringEngagementIsNotAllowed {
-            try completion()
-        }
-    }
-
-    // MARK: - Private
-
-    private func startRootCoordinator(
+    func startRootCoordinator(
```

Comment:
Why is the root coordinator now internal?

Reply:
correct
![image](https://github.com/salemove/ios-sdk-widgets/assets/125041274/04c4347a-7edb-4a35-940a-1f5ef1855b68)